cluster_config.yaml

```diff
@@ -0,0 +1,95 @@
+# The jump host should be in the same network as the cluster.
+# The jump_host_ip should be reachable from the client that runs the cluster install.
+jump_host_ip: ''
```

Comment:
As mentioned, I think we should maybe look to get rid of the jumphost and instead document that the script is intended to be run on a host on the same network as or otherwise with low latency to the cluster nodes.

Reply:
After discussing this with @alexmolev, we decided that it would be best to run the code from the jump host itself and make it simpler.